.circleci/config.yml

```diff
@@ -381,20 +381,22 @@ commands:
           command: |
             echo 'export REDIRECT_URL=$REDIRECT_URL_CONTENT' >> $BASH_ENV
             echo 'export SHOW_DISABLED_INDUSTRIES=true' >> $BASH_ENV
+            echo 'export COGNITO_USER_POOL_ID=$COGNITO_USER_POOL_ID_CONTENT' >> $BASH_ENV
+            echo 'export COGNITO_IDENTITY_POOL_ID=$COGNITO_IDENTITY_POOL_ID_CONTENT' >> $BASH_ENV
+            echo 'export REDIRECT_URL=$REDIRECT_URL_CONTENT' >> $BASH_ENV
 
   set-accessibility-testing-env-vars:
     steps:
       - run:
           name: Set Accessibility Testing Environment Variables
           command: |
+            echo 'export COGNITO_WEB_CLIENT_ID=$COGNITO_WEB_CLIENT_ID_ACCESSIBILITY_TESTING' >> $BASH_ENV
+            echo 'export COGNITO_USER_POOL_ID=$COGNITO_USER_POOL_ID_ACCESSIBILITY_TESTING' >> $BASH_ENV
+            echo 'export COGNITO_IDENTITY_POOL_ID=$COGNITO_IDENTITY_POOL_ID_ACCESSIBILITY_TESTING' >> $BASH_ENV
             echo 'export REDIRECT_URL=$REDIRECT_URL_ACCESSIBILITY_TESTING' >> $BASH_ENV
 
   deploy-content:
     steps:
-      - configure-and-build
-      - setup_remote_docker:
-          version: edge
-          docker_layer_caching: true
```

Comment:
[dust] Do we no longer need to set up Docker before running `package-webapp-docker` later in the script?

Reply:
@seidior So we are still setting up Docker. However because we run it in the deploy job prior to `deploy-content` and `deploy-accessibility-testing`  which is redundant and we get an error thrown as well in the build. See Error below: 
 ```More than one setup_remote_docker is not valid, please adjust in job , , accessibility-testing-deploy,```

Reply:
https://app.circleci.com/pipelines/github/newjersey/navigator.business.nj.gov/27281